### DISCUSSION TITLE
Value slider tooltip on touch devices should be offset more

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#28469] [Ideas]
I find my finger being under the tooltip (or part of over it) every single time I want to change my osu!mania scroll speed. It's really hard to see the exact value of the slider.

**Comment by shiumano** [comment]
To begin with, you can't fiddle with the slider on a touch screen where there is no guarantee that it will be in the same place when you release your finger.
The best thing would be to implement a pop-up that allows direct input of values!

**Reply by ghost** [reply]
Yeah that sounds really optimal, it should also be an option on PC as well...

**Reply by Stoppedpuma** [reply]
Manual input for sliders is already planned, see #24900
